model/common/src/icon4py/model/common/grid/vertical.py

```diff
@@ -226,8 +226,8 @@ def _bottom_level(self, domain: Domain) -> int:
         return self.size(domain.dim)
 
     @property
-    def interface_physical_height(self) -> fa.KField[ta.wpfloat]:
-        return self._vct_a
+    def interface_physical_height(self) -> fa.KField[wpfloat]:
+        return gtx.astype(self._vct_a, wpfloat)
```

Comment:
Just a question: is there are any particular reason why the conversion is done here instead of at initialization in the `post_init` method?